### DISCUSSION TITLE
Why does it sometimes take multiple minutes for a task to kick in?

### DISCUSSION THREAD

**Discussion by philippeboyd** [concourse/concourse#7990] [Help & Support] [answered]
Once my task started and the worker was selected, it took 6 minutes for it to kick in. Can sometimes take 10 minutes. 

These are the first lines of the task's script
```bash
#!/bin/bash -e

apt-get update
apt-get install -y make curl git libxml2-utils
```

![image](https://user-images.githubusercontent.com/3239656/150547048-6d3e2977-7aaf-44cf-9f11-a5c25a124822.png)

I'm trying to figure out why concourse is so slow sometimes.

**Comment by andy-paine** [comment, marked as answer]
More than likely Concourse was streaming a volume (from a `get` step or previous `task`) to the selected worker. A PR just got merged and should be released in v7.7.0 to provide a build event similar to the `selected worker` one to indicate that this is happening https://github.com/concourse/concourse/pull/8031
